.github/workflows/linux.yml

```diff
@@ -142,12 +142,10 @@ jobs:
       env:
```

Comment:
The name of the job should change since it no longer builds a wheel

Reply:
This confused me for a moment - it's the name of the step rather than the name of the job. "Build a wheel" -> "Install as editable".

There's also a comment in the line under the `full` job name that needs a tweak.